src/webui/www/webui.qrc

```diff
@@ -15,9 +18,14 @@
         <file>private/edittracker.html</file>
         <file>private/index.html</file>
         <file>private/newcategory.html</file>
+        <file>private/newfeed.html</file>
+        <file>private/newfolder.html</file>
+        <file>private/newrule.html</file>
         <file>private/newtag.html</file>
         <file>private/rename.html</file>
+        <file>private/rename_feed.html</file>
         <file>private/rename_file.html</file>
+        <file>private/rename_rule.html</file>
```

Comment:
Maybe create a rss folder for them?

Reply:
I think all the dialog html files kind of clutter the directory. Should I move all dialogs to `private/dialogs` and all rss specific dialogs to `private/dialogs/rss`?

Reply:
I think you should limit to the newly added files. It is not a good idea to include too many changes in a PR, especially in one that already has 2000+ lines addition...

Reply:
Do you think in general moving the dialogs to a separate folder is a good idea? I would create a new PR moving them.

Reply:
seems every file except index.html is a dialog, if so, then I don't have strong feelings for it.

Reply:
All html files in the root of the "private" directory are "full" html pages. As @Chocobo1 pointed out, they're mostly dialogs that are displayed in iframes. Anything in the "views" directly is a partial page that's meant to be added to an existing page. I think for now this PR should stick to this same basic principle, and then we can target reorganizing the webui files in a separate PR.